WooCommerce/Classes/ViewModels/Order Details/OrderDetailsViewModel.swift

```diff
@@ -442,17 +442,19 @@ extension OrderDetailsViewModel {
         stores.dispatch(deleteTrackingAction)
     }
 
-    func collectPayment() {
+    func collectPayment(onCompletion: @escaping (Result<Void, Error>) -> Void) {
         // Mock Payment. We will have to instantiate these parameters with
         // the proper amount, currency and readable descripitions later.
+        // We could pull the amount from the order but in test mode the Stripe Terminal SDK fails to process amounts that do not end in 00
         let paymentParameters = PaymentParameters(amount: 100,
```

Comment:
Could we change this to  something like $5 just in case we accidentally do this live - I generally like to test WooCommerce with dollar amounts that wouldn't stress me out if they were to turn out to be live charges.

Reply:
I am not sure I follow. Eventually we are going to have to update this to pass the order's total amount 

[The documentation says](https://stripe.com/docs/terminal/payments#create-payment): 
> Use test amounts to try producing different results. An amount ending in 00 results in an approved payment.

I don't know if that answers your question though?

Reply:
No worries. I'm asking Karl if the amount is in cents and how that might vary by currency code